docs/_docs/reference/changed-features/binary-literals.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: doc-page
+title: "Binary Integer Literals"
+nightlyOf: https://docs.scala-lang.org/scala3/reference/changed-features/binary-integer-literals.html
+---
+
```

Comment:
I think you also need to update the sidebar manually

Reply:
Oh right !